tests/template_tests/syntax_tests/i18n/base.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+from django.conf import settings
+from django.test import SimpleTestCase
+from django.utils._os import upath
+from django.utils.translation import activate, get_language
+
+here = os.path.dirname(os.path.dirname(os.path.abspath(upath(__file__))))
+pdir = os.path.split(os.path.split(os.path.abspath(here))[0])[0]
+extended_locale_paths = settings.LOCALE_PATHS + [
+    os.path.join(pdir, 'i18n', 'other', 'locale'),
+]
+
+
+class BaseTestCase(SimpleTestCase):
```

Comment:
I feel like there could be a better class name here, to indicate that this isn't a generic base test case. Perhaps `I18nTestCase` would do? I realise it's not a big deal since no other tests should import this `base.py`, but it's useful documentation to someone reading the code. There's also precedent for this in e.g. `WidgetTest` and `SiteMapTestsBase`.